README.md

```diff
@@ -46,13 +55,45 @@ JupyterHubSSH is made up of two main components:
 	$ python -m jupyterhub_ssh
 	```
 
-## Instructions on how to use it
+### Kuberbetes based deployment
+If your JupyterHub was deployed using Kubernetes, you can use the Helm charts available in this repo to deploy JupyterHub SSH & SFTP
+directly into your Kubernetes cluster.
+
+	```bash
+	# Let helm the command line tool know about a Helm chart repository
+	# that we decide to name jupyterhub.
+	$ helm repo add jupyterhub https://yuvipanda.github.io/jupyterhub-ssh/
+	$ helm repo update
+
+	# Simplified example on how to install a Helm chart from a Helm chart repository
+	# named jupyterhub-ssh. See the Helm chart's documentation for additional details
+	# required.
+	$ helm install jupyterhub-ssh/<helm chart name> --version <helm chart version>
+	```
```

Comment:
```suggestion
	```bash
	# Let helm the command line tool know about a Helm chart repository
	# that we decide to name jupyterhub.
	$ helm repo add jupyterhub-ssh https://yuvipanda.github.io/jupyterhub-ssh/
	$ helm repo update

	# Simplified example on how to install a Helm chart from a Helm chart repository
	# named jupyterhub-ssh. See the Helm chart's documentation for additional details
	# required.
	$ helm install jupyterhub-ssh/jupyterhub-ssh --version <helm chart version> --set hubUrl=https://jupyter.example.org --set-file hostKey=<path to a private SSH key>
	```
```

I think this is about right. Most people have a helm repository for charts named JupyterHub already, so reusing jupyterhub would be problematic, so let's call this Helm chart repository (in this repo's gh-pages) to jupyterhub-ssh for now.

I also filled in the helm chart name, which is well known and won't change, to jupyterhub-ssh.

I also filled in the two variables I know that is strictly required currently.

@yuvipanda btw, I think we need too have SFTP disabled by default for this Helm chart as we also require a PVC name for it.